src/features/applicationView/offerCard/sendInvoiceForm/SendInvoiceForm.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import * as Yup from 'yup';
+import { useTranslation } from 'react-i18next';
+import { TextInput } from 'hds-react';
+import { Form, Formik } from 'formik';
+
+import styles from './sendInvoiceForm.module.scss';
+import FormHeader from '../../../../common/formHeader/FormHeader';
+import Button from '../../../../common/button/Button';
+import Text from '../../../../common/text/Text';
+
+type FormValues = {
+  dueDate: Date;
+};
+
+export type SendInvoiceFormProps = {
+  email: string | null;
+  isSubmitting: boolean;
+  onCancel: () => void;
+  onSubmit: (data: FormValues) => void;
+};
+
+const SendInvoiceForm = ({ email, onSubmit, onCancel, isSubmitting }: SendInvoiceFormProps) => {
+  const { t } = useTranslation();
+
+  const validationSchema = Yup.object<FormValues>().shape({
+    dueDate: Yup.date(),
+  });
+
+  const initial: FormValues = {
+    dueDate: new Date(Date.now()),
+  };
+
+  return (
+    <Formik
+      initialValues={initial}
+      onSubmit={(values) => onSubmit(values)}
+      validateOnBlur={false}
+      validateOnChange={false}
+      validationSchema={validationSchema}
+    >
+      {({ values, errors, handleChange }) => (
+        <Form className={styles.form}>
+          <FormHeader title={t('forms.sendInvoice.title').toUpperCase()} />
+
+          <p className={styles.instructions}>
+            {t('forms.sendInvoice.instructions.paragraph1', {
+              email: email,
+            })}
+          </p>
+          <p className={styles.instructions}>{t('forms.sendInvoice.instructions.paragraph2')}</p>
+
+          <div className={styles.dueDate}>
+            <TextInput
+              id="dueDate"
+              type="date"
+              value={values.dueDate.toString()}
+              onChange={handleChange}
+              label={t('forms.sendInvoice.dueDate')}
+              invalid={!!errors.dueDate}
+              helperText={errors.dueDate as string}
```

Comment:
I suggest falling back to a common error message instead of casting the type

Reply:
Good comment, this helped me catch an issue with the date typing and validation 👍